com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/commands/abort/AbortFailedProcessCommandFactory.java

```diff
@@ -0,0 +1,26 @@
+package com.sap.cloud.lm.sl.cf.process.flowable.commands.abort;
+
+import org.flowable.common.engine.impl.interceptor.Command;
+import org.flowable.engine.impl.jobexecutor.DefaultFailedJobCommandFactory;
+
+import org.springframework.context.annotation.Lazy;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Named
+@Lazy
+public class AbortFailedProcessCommandFactory extends DefaultFailedJobCommandFactory {
```

Comment:
This has naming problem from the beginning -> AbortProcessFailedJobCommandFactory